Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.h

```diff
@@ -401,6 +402,36 @@ namespace AZ
             {
                 const size_t srgSlot = pipelineLayout->GetSlotByIndex(srgIndex);
                 const ShaderResourceGroup* shaderResourceGroup = bindings.m_srgsBySlot[srgSlot];
+                RootParameterBinding binding = pipelineLayout->GetRootParameterBindingByIndex(srgIndex);
+
+                if (binding.m_bindlessTable.IsValid())
+                {
+                    switch (pipelineType)
+                    {
+                    case RHI::PipelineStateType::Draw:
+                        if (binding.m_bindlessTable.IsValid())
+                        {
+                            GetCommandList()->SetGraphicsRootDescriptorTable(
```

Comment:
Dont we only need to bind it once? If we have multiple SRGs using indirect views will we be binding multiple times?

Reply:
We need to bind the bindless table for each slot that requests it unfortunately. We could change the backend code to reuse slots that can alias but it would complicate the code both here and in the compiler. Since we will likely only have one primary SRG that includes bindless view tables, I opted to keep it simple for now, but we can optimize it later for sure.